src/js/popup.js

```diff
@@ -777,20 +777,30 @@ Logic.registerPanel(P_CONTAINER_INFO, {
       tr.classList.add("container-info-tab-row");
       tr.innerHTML = escaped`
         <td></td>
-        <td class="container-info-tab-title truncate-text" title="${tab.url}" >${tab.title}</td>`;
+        <td class="container-info-tab-title truncate-text" title="${tab.url}" >${tab.title}</td>
+        <td><img src="/img/container-close-tab.svg" width="50%" id="${tab.id}" class="container-close-tab" /></td>`;
```

Comment:
Can we move this into the title row instead of it being it's own cell? The idea is that it would be `display: none;` until the user mouse overs that row, then it becomes visible. This will allow more of the title to be visible as we are pretty limited for space here.

Reply:
Also remove `width=50%` and plae all styles into the css file.

Reply:
Rather than the logic to delete it and needing to use the `id=""` instead you could construct the button string before like this:
```javascript
let closeButton = "";
if (!tab.hiddenState) {
  closeButton = `<img src="/img/container-close-tab.svg" class="container-close-tab" />`;
}
tr.innerHTML = escaped`<td></td><td class="container-info-tab-title truncate-text" title="${tab.url}" >${tab.title}${closeButton}</td>`;
```